k8s/celery-beat.yaml.j2

```diff
@@ -0,0 +1,40 @@
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: {{ CELERY_SCHEDULER_NAME }}
+spec:
+  replicas: {{ CELERY_SCHEDULER_REPLICAS }}
+  selector:
+    matchLabels:
+      app: {{ CELERY_SCHEDULER_NAME }}
+  template:
+    metadata:
+      labels:
+        app: {{ CELERY_SCHEDULER_NAME }}
+    spec:
+      volumes:
+      - name: developer-portal-fs
+        persistentVolumeClaim:
+          claimName: {{ APP_PVC_NAME }}
+      containers:
+        - name: {{ CELERY_SCHEDULER_NAME }}
+          image: "{{ APP_IMAGE }}:{{ APP_IMAGE_TAG }}"
+          imagePullPolicy: {{ APP_IMAGE_PULL_POLICY }}
+          volumeMounts:
+            - name: developer-portal-fs
+              mountPath: {{ APP_MOUNT_PATH }}
```

Comment:
There's a related comment to the need / non-need for this mount over in the my PR: https://github.com/mdn/developer-portal/pull/412/files/9d9c6e1caa97cfc9f46c4d375ba685f05964128b#r336957722